keyboards/bfake/bfake.c

```diff
@@ -38,8 +44,26 @@ void rgblight_set(void) {
     i2c_init();
     i2c_send(0xb0, (uint8_t*)led, 3 * RGBLED_NUM);
 }
+#endif
 
 __attribute__ ((weak))
 void matrix_scan_user(void) {
-    rgblight_task();
+#ifdef RGBLIGHT_ENABLE
+	rgblight_task();
```

Comment:
There's an error in some other files, but this shouldn't be called in this file - [it's already called here](https://github.com/qmk/qmk_firmware/blob/4549b0417e7ac99bff36f130f3d7642ec24c2956/tmk_core/protocol/lufa/lufa.c#L1203).

Reply:
That was in the original bfake code; I just added the \#ifdef guard around it. But I updated my commit to remove it. Does the rest look reasonable?